test/example-app/index.js

```diff
@@ -542,10 +594,12 @@ const MovieSearch = {
       }
     ],
     legacyProperties: {
-      // The URLs miss an 's' at the end on purpose for scripting to fix it
-      url: 'https://auth-json-server.zapier.ninja/movie',
-      inputFieldsUrl: 'https://auth-json-server.zapier.ninja/input-field',
-      outputFieldsUrl: 'https://auth-json-server.zapier.ninja/output-field'
+      // The URLs miss an 's' at the end of the resource names on purpose for
```

Comment:
-> `The url misses an 's'...`

Reply:
The comment actually refers to all the URLs below it. Made it clearer in a5dc2a0.